src/julia-syntax.scm

```diff
@@ -562,6 +562,9 @@
              ,(if (any kwarg? pargl) (gensy) UNUSED)
              (call (core kwftype) ,ftype)) ,kw ,@pargl ,@vararg)
           `(block
+            ;; propagate method metadata to keyword sorter
+            ,@(map propagate-method-meta (filter meta? prologue))
```

Comment:
Should filter out expressions that end up empty, `(meta)`.

Reply:
`(filter meta? prologue)` should already filter out the empty `(meta)` case?
https://github.com/JuliaLang/julia/blob/a4e275bd961347d2585b0bebff16f90488df9dd2/src/ast.scm#L527